imports/ui/side-menu/side-menu.jsx

```diff
@@ -66,21 +66,23 @@ class SideMenu extends Component {
         })}
         <Divider />
         {this.linkDrawerItem({
-          href: 'https://unee-t.com/contact-support/',
+          href: 'mailto:support@unee-t.com?subject=' + window.location.href + '&body=' + encodeURIComponent(`user: ${user.emails[0].address}
```

Comment:
Having the email body inlined inside an `encodeURIComponent` call, inlined in a query string parm, inlined in an object sent to a method, inlined inside a JSX snippet... is a bit confusing to read :sweat_smile: 

Can you extract the email body to a separate const that you define earlier or something and use it instead?